doc/source/google/step-zero-gcp.rst

```diff
@@ -55,23 +55,7 @@ your google cloud account.
 
             gcloud components install kubectl
 
-4. Decide and configure to use a specific data center.
-
-   From a terminal, enter:
-
-   .. code-block:: bash
-
-      # Enter a zone
-      ZONE=us-east1-c
-
-      gcloud config set compute/zone $ZONE
-   
-   * ZONE specifies which data center to use. Pick something `from
-     this list
-     <https://cloud.google.com/compute/docs/regions-zones/regions-zones#available>`_.
-     that is not too far away from your users.
-
-5. Create a managed Kubernetes cluster and a default node pool.
```

Comment:
It would be used if we create another node pool, or do various gcloud commands in the future, but it is a "prepare for future" part that adds complexity and yepp let's reduce the complexity.

Reply:
Yeah, I think the 'wow' factor of 'I ran all these commands and they just worked' is very important. Most folks don't need to create multiple pools ever, so we should focus on the default use case.